Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'rugged', git: 'https://github.com/libgit2/rugged.git', submodules: true
+gem 'rubocop'
```

Comment:
Can these be moved to the gemspec? And is there a requirement for rugged from git vs gem?